src/zepben/evolve/streaming/grpc/connect.py

```diff
@@ -29,34 +29,41 @@ def _secure_grpc_channel_builder(host: str = "localhost", rpc_port: int = 50051)
         .make_secure()
 
 
-def _grpc_channel_builder_from_password(client_id: str, username: str, password: str, host: str, rpc_port: int) -> GrpcChannelBuilder:
-    # noinspection PyArgumentList
-    token_fetcher = ZepbenTokenFetcher(
-        audience="https://evolve-ewb/",
-        issuer_domain="zepben.au.auth0.com",
-        auth_method=AuthMethod.AUTH0,
-        verify_certificate=True,
-        token_request_data={
-            'client_id': client_id,
-            'username': username,
-            'password': password,
-            'grant_type': 'password',
-            'scope': 'offline_access'
-        }
-    )
-
+def _grpc_channel_builder_from_password(client_id: str, username: str, password: str, host: str, rpc_port: int, token_fetcher: ZepbenTokenFetcher
+                                        ) -> GrpcChannelBuilder:
+    token_fetcher.token_request_data.update({
+        'client_id': client_id,
+        'username': username,
+        'password': password,
+        'grant_type': 'password',
+        'scope': 'offline_access'
+    })
     return GrpcChannelBuilder() \
         .socket_address(host, rpc_port) \
         .make_secure() \
         .token_fetcher(token_fetcher)
 
 
-def connect_secure(host: str = "localhost", rpc_port: int = 50051) -> grpc.aio.Channel:
+def connect_tls(host: str = "localhost", rpc_port: int = 50051) -> grpc.aio.Channel:
     return _secure_grpc_channel_builder(host, rpc_port).build()
 
 
-def connect_with_password(client_id: str, username: str, password: str, host: str = "localhost", rpc_port: int = 50051) -> grpc.aio.Channel:
-    return _grpc_channel_builder_from_password(client_id, username, password, host, rpc_port).build()
+def connect_with_password(client_id: str, username: str, password: str, host: str = "localhost", rpc_port: int = 50051,
```

Comment:
Can you also add the relevant docs from the `_conn` function for this? Wouldn't hurt to do this for all functions you've added I guess.

Reply:
Actually don't worry I'll get James to do it.